crates/holochain/tests/multi_conductor.rs

```diff
@@ -91,6 +91,7 @@ async fn multi_conductor() -> anyhow::Result<()> {
 
 #[tokio::test(flavor = "multi_thread")]
 #[cfg(feature = "test_utils")]
+#[ignore = "I'm not convinced this test is actually adding value and worth fixing right now"]
 async fn invalid_cell() -> anyhow::Result<()> {
```

Comment:
What is this testing and why is it called `invalid_cell` anyway?

Reply:
Yeh I almost just outright removed it. Maybe I should?
It was meant to find the "invalid cell" error that was seen in production but all it really does is show that we can't reproduce that in our tests.
There might be some value in that but it's definitely not worth the energy right now. I could go either way

Reply:
I'm going to keep this in for now because it would be nice to clean it up and get it working at some point but also happy to remove if anyone feels like that's the best idea